documentation/asciidoc/computers/raspberry-pi/boot-eeprom.adoc

```diff
@@ -1,6 +1,6 @@
 == Raspberry Pi boot EEPROM
 
-Raspberry Pi 5, Raspberry Pi 4, 400, Compute Module 4, and Compute Module 4S computers use an EEPROM to boot the system. All other models of Raspberry Pi computer use the `bootcode.bin` file located in the boot filesystem.
```

Comment:
I think we generally don't mention unreleased products in our documentation? But this is obviously @nathan-contino 's call to make.
(also, should probably add CM5 to the list after CM4S?)

Reply:
Too late for that. Although we should generally refer to products as "Raspberry Pi 5 family" or "Raspberry Pi 4 family" rather than tediously enumerating all products everywhere

Reply:
Fixed via suggestion.